src/EndevorController.ts

```diff
@@ -211,8 +212,17 @@ export class EndevorController {
         });
         this.updateIDs(connName);
       } else {
-        const endevorDataProvider = new EndevorDataProvider();
-        endevorDataProvider.addSession(connName);
+        try {
+          const endevorDataProvider = new EndevorDataProvider();
+          endevorDataProvider.addSession(connName);
+        } catch (error) {
+          logger.info(error.message);
+          logger.trace(
+            'You have an Endevor connection that refer to a non-existent profile named ' +
+              connName +
+              '. To resolve this, you can remove the connection with this profile name in your user settings.'
+          );
+        }
```

Comment:
I think the try/catch block should only be around `endevorDataProvider.addSession(connName);`
Also regarding logging, it can be just want message, like:
`logger.warning(<message_for_user>, <message_for_output_channel>)`

I think the message from `error.message` can go to the user, since it comes from `Profiles.loadNamedProfile()`. The other one can go as a second argument.

Just FYI, the logger we have now has the following behavior:

logger.trace() - only writes to output console
logger.info()
logger.warn()
logger.error() - they all take 2 args. 1st one is for the user and will create appropriate notification window. 2nd one goes to console (should be the trace or detailed message). The logger also adds `(see OUTPUT - ${OUTPUT_CHANNEL_NAME} for more details)` to the message that goes to the notification box, to direct user's attention to the console for further debugging.

Reply:
@Alexandru-Dumitru I don't see anything written in the the OUTPUT. Maybe I'm doing something wrong.

![image](https://user-images.githubusercontent.com/45042840/100604681-272d4d00-3307-11eb-8a3b-a342bf8a20ef.png)

Reply:
@jellypuno You have to select the correct output channel. There is a drop-down on the right side of the panel. You should be able to find "Explorer for Endevor" there.

Reply:
found it. thank you @Alexandru-Dumitru

Reply:
These changes were made.